tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml

```diff
@@ -209,21 +211,23 @@
     fail_msg: "Unable to detach iscsi client using initiator name"
     success_msg: "ISCSI client detached successfully using initiator name"
 
-- name: List all clients attached to VG to verify detachment
-  ntnx_volume_groups_iscsi_clients_info_v2:
-    volume_group_ext_id: "{{ vg1_uuid }}"
-  register: result
+# Need to verify detachment
```

Comment:
Remove it. Use VG info to check if the attached client is not present in the VG info.

Reply:
VG info doesnt have attachments info. Ignore